client/src/components/SocialMediaShare.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import {
+  EmailShareButton,
+  FacebookShareButton,
+  FacebookMessengerIcon,
+  FacebookMessengerShareButton,
+  TelegramShareButton,
+  LinkedinShareButton,
+  TwitterShareButton,
+  TumblrShareButton,
+  WhatsappShareButton,
+  EmailIcon,
+  FacebookIcon,
+  LinkedinIcon,
+  TelegramIcon,
+  TumblrIcon,
+  TwitterIcon,
+  WhatsappIcon
+} from 'react-share'
+import './SocialMediaShare.css'
+
+function SocialMediaShare ({ item }) {
+  console.log('SocialMediaShare -> item', item)
```

Comment:
I can imagine this is a console.log for debugging so now you don't need it anymore :)